ethstats/ethstats.go

```diff
@@ -567,9 +555,8 @@ type nodeInfo struct {
 
 // authMsg is the authentication infos needed to login to a monitoring server.
 type authMsg struct {
-	ID      string         `json:"id"`
-	Address common.Address `json:"address"`
-	Info    nodeInfo       `json:"info"`
+	ID   string   `json:"id"`
```

Comment:
Is this change backwards compatible?

Reply:
yes it is, celostats was using this field since a couple of days ago. I removed it there first now it can be removed here. I don't know any more usages of this field.

Reply:
Would a geth instance still sending `ID, Address, Info` still be able to authenticate with the updated celostats?

Reply:
yes